app/controllers/users_controller.rb

```diff
@@ -1,16 +1,17 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:index, :edit, :update, :show]
+  before_action :correct_user, only: [:edit, :update]
+  before_action :find_user, except: %i(index new create)
+
+  def index
+    @pagy, @users = pagy(User.all, items: 3)
```

Comment:
- `User.all` -> viết scope sắp xếp bên model User em nhé, khi get list thì nên xử lý trc khi hiển thị
- `3` -> cho hằng số vào file settings
```suggestion
    @pagy, @users = pagy User.all, items: 3
```